recipes/TextToSpeech.py

```diff
@@ -57,7 +59,7 @@ class TextToSpeechSettings(BaseModel):
 
     openai_voice_name: OpenAI_TTS_Voices.api_choices | None
     openai_tts_model: OpenAI_TTS_Models.api_choices | None
-
+    ghana_tts_language: GHANA_TTS_LANGUAGES.api_choices | None
```

Comment:
Please change to `ghana_nlp_tts_language` given "ghana tts" doesnt show the right thing when you google vs "ghana nlp tts" and also for consistency with the rest of the codebase that uses `ghana_nlp_*` prefix

Reply:
@devxpy updated